config/samples_2016APV.yaml

```diff
@@ -1,5 +1,17 @@
 GLOBAL:
-period: '2016APV'
+  era: '2016_HIPM'
+  luminosity: 36.31
```

Comment:
this is full 2016 lumi - better not to put

Reply:
ok